src/System.Net.Sockets/src/System/Net/Sockets/BaseOverlappedAsyncResult.Unix.cs

```diff
@@ -23,7 +23,7 @@ public BaseOverlappedAsyncResult(Socket socket, Object asyncState, AsyncCallback
             if (NetEventSource.IsEnabled) NetEventSource.Info(this, socket);
         }
 
-        protected void CompletionCallback(int numBytes, SocketError errorCode)
+        internal void CompletionCallback(int numBytes, SocketError errorCode)
```

Comment:
The issue is, generally it would be very bad to accidentally call CompletionCallback on the base class instead of the derived one.  In this case it doesn't matter because we just defer to the base class.  But I do think it's worth maintaining the same pattern here as everywhere else.